docs/ovs_communication.adoc

```diff
@@ -43,12 +43,32 @@ TBD - XiaoDong
 
 How do we call into OVS, what is the exact command we use to program a new port? What is the port configuration info needed?
 
+A: we would use ovs-vsctl add-port cmd with lots of options to add or to configure (set port)
 
 == OVS-db entries on Local Host
 
 TBD - XiaoDong, I think it will be good to show how an entry looks like in ovs-db here
 
-
+A: here is one example:
```

Comment:
adoc doesn't align the below text nicely. We may need to do the below to make it look nice.

[source,c++]
------------------------------------------------------------
text/code stuff
------------------------------------------------------------

Reply:
e72c1206-4e56-47fc-83b9-5c986af5ba8a
    Bridge "ovs0"
        Port "b8afb0bd725b4_l"
            tag: 100
            Interface "b8afb0bd725b4_l"
        Port "2fe20f38124d4_l"
            tag: 100
            Interface "2fe20f38124d4_l"
        Port "4a41eb1cac8c4_l"
            tag: 200
            Interface "4a41eb1cac8c4_l"
        Port "ovs0"
            Interface "ovs0"
                type: internal
        Port "vxlan1"
            Interface "vxlan1"
                type: vxlan
                options: {key=flow}
    ovs_version: "2.9.5"